homeassistant/components/unifiprotect/sensor.py

```diff
@@ -346,6 +355,14 @@ class ProtectSensorEntityDescription(ProtectRequiredKeysMixin, SensorEntityDescr
     ),
 )
 
+MOTION_SENSORS: tuple[ProtectSensorEntityDescription, ...] = (
+    ProtectSensorEntityDescription(
+        key=_KEY_OBJECT,
+        name="Detected Object",
+        device_class=DEVICE_CLASS_OCCUPANCY,
```

Comment:
This doesn't look like a valid sensor device class.
We should use the SensorDeviceClass Enum for device classes